00-prep/lab/1-ES6-practice/let-const/js/app.js

```diff
@@ -128,7 +127,7 @@ if(localStorage.busmall){
   allProducts = JSON.parse(localStorage.busmall);
 } else {
   console.log('There is no local storage data; initialize app by creating instances');
-  for(var i = 0; i < names.length; i++) {
+  for(const i = 0; i < names.length; i++) {
```

Comment:
const should not be used in a for loop. You won't be able to increment because it is constant.